README.md

```diff
@@ -6,6 +6,97 @@ channel branches (e.g. `channels/8/edge`).
+
+New Relic 2022 [Java Ecosystem report](https://newrelic.com/resources/report/2022-state-of-java-ecosystem) listed following top JDK vendors:
+ - Oracle Corporation 34.48%
+ - Amazon 22.04%
+ - Eclipse Adoptimum 11.48%
+ - Azul Zulu build of OpenJDK 16%
+
+Snyk 2021 [JVM Ecosystem](https://snyk.io/jvm-ecosystem-report-2021/) had the following breakdown:
+ - Eclipse Adoptimum 44%
+ - Oracle Corporation OpenJDK 28%
+ - Oracle Corporation JDK 23%
+ - Azul Zulu build of OpenJDK 16%
+ - Amazon Coretto 8%
+
+The differences in the vendor distribution could be attributed to the audience providing survey responses.
+
+The chiselled JRE container is built based on Ubuntu 22.04 version of Java 8 runtime - `8u352-b08`.
+
+This section provides a comparison with readily-available JRE 8 images for Ubuntu 22.04:
+ - Eclipse Adoptium: `eclipse-temurin:8u352-b08-jre-jammy`
+ - Amazon: `amazoncorretto:8u352-alpine3.14-jre`
+
+Azul Zulu does not provide a JRE image: https://hub.docker.com/r/azul/zulu-openjdk and it was not evaluated.
+Oracle does not provide official image of the Java Runtime Environment 8 and it was not evaluated.
+
+### Image size
+
+|Image|Tag|Uncompressed Size| Compressed Size|
+|-----|---|----| ----------------------------|
+| eclipse-temurin|8u352-b08-jre-jammy|221MB|80M|
+| amazoncorretto| 8u352-alpine3.14-jre| 110MB | 41M|
+| ubuntu/chiselled-jre|8-22.04_edge| 123MB|46M |
+
+
+The major points of difference with Temurin image are:
+- `/bin` and `/usr/bin` are removed, which occupy 20MB (compressed) in Temurin
+- `/var` is removed, which occupies 7.7MB due to `dpkg`
+- only minimal set of libraries is present in /usr/lib/x86_64-linux-gnu, saving 39M
+- contents of /usr/share are not present (31MB), assuming that for things like local time zone information, it is either mapped into the container, or containers run in GMT.
+
+The major points of difference with Corretto image are:
+ - Corretto deploys busybox as a shell
+ - Corretto does not have fontconfig/fonts libraries. This causes `java.awt.Font.createFont()` to fail with `java.lang.UnsatisfiedLinkError`.
+ - Corretto does not provide `glib` libraries imported by the URL proxy selector. `java.net.ProxySelector.getDefault()` call will always return `Direct Proxy`.
+
+The JRE differences itself are minimal. Th chiselled image removes libawt_xawt.so and libsplashscren.so along with accessibility support. Executables, except `java`, are removed from `jre/bin`.
+Note: chiselled docker at the moment does not provide classes.jsa (Class Data Cache) in line with Temurin JRE and it has to be generated.
+
+Below are image sizes of the deployed `acmeair` benchmark application
+|Image|Base Image|Uncompressed Size| Compressed Size|
```

Comment:
I find this table quite hard to read and compare. I guess the easiest solution would be to split in three tables, one for each "platform" (websphere, springboot, tomcat).